data/json/furniture_and_terrain/furniture-storage.json

```
@@ -433,7 +433,7 @@
         { "item": "nail", "charges": [ 32, 40 ] }
       ]
     },
-    "max_volume": "1500 L",
+    "max_volume": 7000,
```

You're increasing the volume here.
```suggestion
    "max_volume": 6000,
```

---

Then it'd just be kept the same as before change, no non-metric strings for you @anothersimulacrum 😛